### ISSUE TITLE
Change in ansible-lint output format at 5.x breaks regex

### ISSUE THREAD

**Issue by piersharding** [wasilak/ansible-lint-junit#10]
Hi - 
It appears that the output format generated by ansible-lint  for pep8 has changed so that the regex (https://github.com/wasilak/ansible-lint-junit/blob/master/bootstrap/bootstrap.py#L49)  that parses the lines no longer works as expected:
old output:
```
playbooks/roles/ceph/tasks/main.yml:18: [E403] Package installs should not use latest
```
New style:
```
playbooks/openvpn_server.yml:14: unnamed-task All tasks should be named
```
changing the regex to the following appears to correct for this:
```
line_regex = re.compile('^(.*?):(\\d+?):\\s(.*)\\s(.*)$')
```
Would it be possible to update to this, or is there another way of solving this?
Thanks,
Piers Harding.

**Comment by wasilak** [comment]
Hi @piersharding, I think I'v resolved it, please check #11 

Cheers!

**Comment by piersharding** [comment]
That's great thanks!
